accname/index.html

```diff
@@ -635,8 +635,12 @@ <h4>Computation steps</h4>
                       <span id="step2F.ii"><!-- Don't link to this legacy numbered ID. --></span><em>Name From Generated Content:</em> Check for
                       <abbr title="Cascading Style Sheets">CSS</abbr> generated textual content associated with the <code>current node</code> and include it in the <code>accumulated text</code>. The
                       <abbr title="Cascading Style Sheets">CSS</abbr> <a href="https://www.w3.org/TR/CSS2/generate.html#before-after-content"><code>::before</code> and <code>::after</code></a> pseudo
-                      elements [[!CSS2]] can provide textual content for [=element|elements=] that have a content model.
+                      elements [[!CSS2]] can provide textual content for [=element|elements=] that have a content model, in addition to [=element|elements=] that support the <a href="https://www.w3.org/TR/css-pseudo-4/#selectordef-marker"><code>::marker</code> pseudo element if it is visually rendered.
                       <ol>
+                        <li id="comp_name_from_content_pseudo_element_marker">
+                          For <code>::marker</code> pseudo elements, <a class="termref">User agents</a> MUST append <abbr title="Cascading Style Sheets">CSS</abbr> textual content, without a space,
+                          to the textual content of the <code>current node</code> if the <code>::marker</code> is visually rendered and the <code>current node</code> supports <code>::marker</code>.
```

Comment:
just not sure about "visually rendered" here, per my previous comment.  but otherwise think this is fine.